rules/no-empty-file.js

```diff
@@ -6,7 +6,7 @@ const messages = {
 	[MESSAGE_ID]: 'Empty files are not allowed.',
 };
 
-const isDirective = node => node.type === 'ExpressionStatement' && 'directive' in node;
+const isDirective = node => node.type === 'ExpressionStatement' && node.directive !== undefined;
```

Comment:
```suggestion
const isDirective = node => node.type === 'ExpressionStatement' && typeof node.directive === 'string';
```

Just in case parser change it to `null` in future.